app/models/card.rb

```diff
@@ -12,7 +12,9 @@ class Card < ActiveRecord::Base
 
   validate :the_text_cannot_be_the_same
 
-  scope :pending, -> { where('review_date <=?', Date.today).order("RANDOM()") }
+  scope :pending, -> {
+    where('review_date <=?', DateTime.now).order("RANDOM()")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.